README.md

```diff
@@ -152,12 +165,12 @@ This section covers how to test the application with SoftHSM.
     The token has been initialized and is reassigned to slot 824959035
     ```
 
-4.  Set environment variables. The CRYPTOKI_SLOT_ID should be set to the reassigned slot id from the second step. The CRYPTOKI_USER_PIN should be set to the user PIN from the second step.
-    ```bash
-    export QRYPT_TOKEN=qrypttokenfromportal
-    export CRYPTOKI_LIB=/usr/local/lib/softhsm/libsofthsm2.so
-    export CRYPTOKI_SLOT_ID=824959035
-    export CRYPTOKI_USER_PIN=1234
+4.  Create or update the configuration file at /etc/qseed/qseed_config.yml. The cryptoki_slot_id should be set to the reassigned slot id from the second step. The cryptoki_user_pin should be set to the user PIN from the second step.
+    ```yaml
+    qrypt_token: qrypttokenfromportal
```

Comment:
Andrew - just to check, we don't want size & period here, right?

Reply:
Correct, they will default to 48 and 10 if not included.